CHANGELOG.md

```diff
@@ -5,7 +5,25 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 2.30.0 (in progress)
 ========================
 
-TODO(lrivas): Write public API to support VDI.
+New Features
+------------
+
+### WebRTC API Overrides (Beta)
+
+The SDK now enables you to override WebRTC APIs with the following options. If your environment supports WebRTC redirection—such as [Citrix HDX](https://www.citrix.com/solutions/vdi-and-daas/hdx/what-is-hdx.html)'s WebRTC [redirection technologies](https://www.citrix.com/blogs/2019/01/15/hdx-a-webrtc-manifesto/)—your application can leverage this new beta feature to enhance media quality in those environments.
+
+**ConnectOptions**
```

Comment:
It might be better to link to the developer docs page here. Let me send you a link to the developer docs page for your review.

Reply:
Done!